bosh-setup/azuredeploy.json

```diff
@@ -113,10 +113,10 @@
 
     "baseUriAzureCloud": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/bosh-setup/",
```

Comment:
We want to have templates that work well for developers as well as end-users, showing how templates can be flexible and used across clouds with "standard" tools and scripts.  You can default the parameter values as described in the bp guide and portal users won't need to worry about it.